app/controllers/communities_controller.rb

```diff
@@ -13,6 +13,7 @@ def index
     @communities = Community.where('name ilike ?', "%#{@search_text}%") if search_params.present?
 
     redirect_to community_path(@communities.first) if @communities.count == 1
+    @communities = @communities.order(:name).page(params[:page]).per(24)
```

Comment:
I've been waffling back and forth in my head on this, but I think I'm leaning toward the opinion that if we're going to change the _default_ paging, it should probably be done in the model (or site wide if that's more appropriate).    

18 might be a better number too, 24 seems a little long.  It goes well off my display and my display has a pretty high resolution.